external/webserver/http_client.c

```diff
@@ -290,15 +319,17 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 					HTTP_LOGD("[BODY_END]buf_len : %d len->sentence_start : %d len->message_len : %d len->content_len %d\n", buf_len, len->sentence_start, len->message_len, len->content_len);
 					//HTTP_LOGD("All body readed : \n%s\n", *body);
 
-					response->total_len = len->content_len;
+					if(response) {
```

Comment:
Please leave spaces as per the coding rules. "if(response){" --> "if (response) {"

Reply:
done